### ISSUE TITLE
Typo in Quickstart Guide?

### ISSUE THREAD

**Issue by Cleod9** [kennethreitz/requests#3672]
Reference: https://github.com/kennethreitz/requests/blob/master/docs/user/quickstart.rst#cookies
```
>>> jar.set('tasty_cookie', 'yum', site='httpbin.org', path='/cookies')
>>> jar.set('gross_cookie', 'blech', site='httpbin.org', path='/elsewhere')
```
Running this example I get the following error:
`TypeError: create_cookie() got unexpected keyword arguments: ['site']`

Shouldn't `site=` be `domain=`? The error goes away if I try that. If so I'll be glad to make a PR, I just want to make sure I'm not missing something

**Comment by nateprewitt** [comment]
Good spot, @Cleod9! `domain` is definitely the keyword we want here. Would you be interested in opening a PR fixing this?

**Comment by nateprewitt** [comment]
Good spot, @Cleod9! `domain` is definitely the keyword we want here. A PR for this would be very welcome.

**Comment by Cleod9** [comment]
No problem, I should be able to get a PR up within the next couple days

**Comment by Cleod9** [comment]
PR is up :)

**Comment by Lukasa** [comment]
Resolved by #3691.
